GoToProject/src/containers/AddNewSite/index.js

```diff
@@ -9,7 +9,16 @@ import { addNewSite } from '../../library/constants/validationSchema';
 import { newSite } from '../../library/constants/dataForm';
 import { Inputs } from '../../library/constants/methods';
 
-export const AddNewSite = () => {
+export const AddNewSite = ({navigation}) => {
+
+  /* at the moment it is only for testing */
+  const handleNewSite = (values) => {
+    const {siteName, phone, categories} = values;
+    console.log(`siteName: ${siteName}`);
+    console.log(`phone: ${phone}`);
+    console.log(`categories: ${categories}`);
+    navigation.navigate('WorkShedules');
```

Comment:
Avoid `console.log`

Reply:
I'll take it into account for now it's just tests